db/dispatcher.go

```diff
@@ -122,7 +126,8 @@ func getKey(event core.Event) (key datastore.Key, err error) {
 	}
 	time := strconv.FormatInt(unix, 10)
 	key = dsDispatcherPrefix.ChildString(time).
-		ChildString(event.InstanceID().String()).
-		ChildString(event.Collection())
+		ChildString(event.Collection()).
```

Comment:
Up for discussion, but it defs makes queries for specific collection types within a db easier, and a lot more efficient.

Reply:
Nice, good call